src/main/java/org/kuali/kfs/sys/service/impl/GeneralLedgerPendingEntryServiceImpl.java

```diff
@@ -0,0 +1,1081 @@
+        bankOffsetEntry.setReferenceFinancialSystemOriginationCode(null);
+
+        if (StringUtils.isBlank(bank.getCashOffsetSubAccountNumber())) {
+            bankOffsetEntry.setSubAccountNumber(KFSConstants.getDashSubAccountNumber());
+        } else {
+            SubAccount cashOffsetSubAccount = bank.getCashOffsetSubAccount();
+            if (ObjectUtils.isNull(cashOffsetSubAccount)) {
+                GlobalVariables.getMessageMap().putError(errorPropertyName,
+                        KFSKeyConstants.ERROR_DOCUMENT_BANK_OFFSET_NONEXISTENT_SUB_ACCOUNT, bank.getBankCode(),
+                        cashOffsetAccount.getChartOfAccountsCode(), cashOffsetAccount.getAccountNumber(),
+                        bank.getCashOffsetSubAccountNumber());
+                return false;
+            }
+
+            if (!cashOffsetSubAccount.isActive()) {
+                GlobalVariables.getMessageMap().putError(errorPropertyName,
+                        KFSKeyConstants.ERROR_DOCUMENT_BANK_OFFSET_INACTIVE_SUB_ACCOUNT, bank.getBankCode(),
+                        cashOffsetAccount.getChartOfAccountsCode(), cashOffsetAccount.getAccountNumber(),
+                        bank.getCashOffsetSubAccountNumber());
+                return false;
+            }
+
+            bankOffsetEntry.setSubAccountNumber(bank.getCashOffsetSubAccountNumber());
+        }
+
+        if (StringUtils.isBlank(bank.getCashOffsetSubObjectCode())) {
+            bankOffsetEntry.setFinancialSubObjectCode(KFSConstants.getDashFinancialSubObjectCode());
+        } else {
+            SubObjectCode cashOffsetSubObject = bank.getCashOffsetSubObject();
+            if (ObjectUtils.isNull(cashOffsetSubObject)) {
+                GlobalVariables.getMessageMap().putError(errorPropertyName,
+                        KFSKeyConstants.ERROR_DOCUMENT_BANK_OFFSET_NONEXISTENT_SUB_OBJ, bank.getBankCode(),
+                        cashOffsetAccount.getChartOfAccountsCode(), cashOffsetAccount.getAccountNumber(),
+                        cashOffsetObject.getFinancialObjectCode(), bank.getCashOffsetSubObjectCode());
+                return false;
+            }
+
+            if (!cashOffsetSubObject.isActive()) {
+                GlobalVariables.getMessageMap().putError(errorPropertyName,
+                        KFSKeyConstants.ERROR_DOCUMENT_BANK_OFFSET_INACTIVE_SUB_OBJ, bank.getBankCode(),
+                        cashOffsetAccount.getChartOfAccountsCode(), cashOffsetAccount.getAccountNumber(),
+                        cashOffsetObject.getFinancialObjectCode(), bank.getCashOffsetSubObjectCode());
+                return false;
+            }
+
+            bankOffsetEntry.setFinancialSubObjectCode(bank.getCashOffsetSubObjectCode());
+        }
+
+        bankOffsetEntry.setTransactionEntryOffsetIndicator(true);
+        bankOffsetEntry.setTransactionLedgerEntryAmount(depositAmount.abs());
+        bankOffsetEntry.setUniversityFiscalPeriodCode(financialDocument.getPostingPeriodCode());
```

Comment:
Does this line also need to be updated to set the fiscal period to null? I believe this line had the null-value setup prior to our 10/29/2020 financials upgrade.

Reply:
This line wasn't changed in recent history, only the line were I commented the CU customization was changed.  Also, the kfs-dev validation only validated that one change.  I don't think we should add changes.

Reply:
Yes, I double checked the kfs-dev version, this line exists there, and testing didn't find a problem

Reply:
Ok, thanks for checking. I'm fine with moving this change forward as-is, but we'll need the functionals to verify that the bank offset entries are being posted to the correct fiscal period. We may need a follow-up PR if this line does end up needing a change.

Reply:
agreed